apps/storybook/src/Tile.stories.tsx

```diff
@@ -197,6 +204,46 @@ Actionable.args = {
   moreOptions: undefined,
 };
 
+export const AnchorLink: Story<TileProps> = (props) => {
+  const {
+    name,
+    description = 'If you click on this stadium, it is going to be selected.',
+    metadata,
+    badge,
+    thumbnail,
+    isNew,
+    isActionable = true,
+    moreOptions,
+    ...rest
+  } = props;
+
+  return (
+    <LinkBox>
+      <Tile
+        name={
+          <LinkOverlay href='/?path=/docs/core-tile--anchor-link'>
+            {name}
+          </LinkOverlay>
+        }
+        description={description}
+        metadata={metadata}
+        badge={badge}
+        thumbnail={thumbnail}
```

Comment:
when using arrow key navigation in screen readers, the contents of the thumbnail area (badges, quick options, img alt) will be announced first.

in https://inclusive-components.design/cards/#alternativetext, the suggestion is to use flex order to tackle this problem. can we try that?